deploy/orioledb/config/etcd-serve-config.tpl

```diff
@@ -0,0 +1,18 @@
+{{- $clusterName := $.cluster.metadata.name }}
+{{- $namespace := $.cluster.metadata.namespace }}
+{{- $orioledb_etcd_from_service_ref := fromJson "{}" }}
+{{- if index $.component "serviceReferences" }}
+  {{- range $i, $e := $.component.serviceReferences }}
+    {{- if eq $i "etcdService" }}
+      {{- $orioledb_etcd_from_service_ref = $e }}
+      {{- break }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+{{- $etcd_server := printf "etcd-etcd:2379" }}
```

Comment:
There should not be a default etcd-server setting here. The form of orioledb needs to be explicitly defined. If serviceRef is not implemented on the cluster, it should throw an error directly. If a default etcd is to be provided, it should be provided at a higher level, such as in Helm, rather than hard-coding a default implementation here.